src/integrationTest/java/org/opensearch/test/framework/cluster/ClusterManager.java

```diff
@@ -52,15 +52,17 @@
 public enum ClusterManager {
 
 	//3 nodes (1m, 2d)
-	DEFAULT(new NodeSettings(true, false), new NodeSettings(false, true), new NodeSettings(false, true)),
+	DEFAULT(new NodeSettings(true, false, false), new NodeSettings(false, true, false), new NodeSettings(false, true, false)),
 
 	//1 node (1md)
-	SINGLENODE(new NodeSettings(true, true)),
+	SINGLENODE(new NodeSettings(true, true, false)),
```

Comment:
Interpreting these configurations is hard, what do you think of having an enum for NodeRoles with values like, ClusterManager, Data, RemoteCluster, and then just accepting a list for the node settings.  

E.g.
```java
SINGLENODE(List.of(CLUSTER_MANAGER, REMOTE_CLUSTER)),
```

Reply:
done